zone/client_packet.cpp

```diff
@@ -7712,6 +7739,14 @@ void Client::Handle_OP_GuildBank(const EQApplicationPacket *app)
 			break;
 		}
 
+		if (!guild_mgr.CheckPermission(GuildID(), GuildRank(), GUILD_ACTION_BANK_WITHDRAW_ITEMS))
+		{
+			Message(Chat::Red, "You do not have permission to withdraw.");
```

Comment:
Is there an eqstr for this? Or is this a custom fallback?

Reply:
6107 You do not have permission to withdraw that item.